www/nginx/src/opnsense/mvc/app/controllers/OPNsense/Nginx/Api/LogsController.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/*
+
+    Copyright (C) 2018 Fabian Franz
+    All rights reserved.
+
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are met:
+
+    1. Redistributions of source code must retain the above copyright notice,
+       this list of conditions and the following disclaimer.
+
+    2. Redistributions in binary form must reproduce the above copyright
+       notice, this list of conditions and the following disclaimer in the
+       documentation and/or other materials provided with the distribution.
+
+    THIS SOFTWARE IS PROVIDED ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES,
+    INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY
+    AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY,
+    OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+    SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+    INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+    CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+    ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+namespace OPNsense\Nginx\Api;
+
+use OPNsense\Base\ApiControllerBase;
+use OPNsense\Core\Backend;
+use OPNsense\Nginx\Nginx;
+
+
+class LogsController extends ApiControllerBase
+{
+    private $nginx;
+    public function accessesAction($uuid = null) {
+        $this->nginx = new Nginx();
+        // emulate rest API -> /accesses delivers a list of servers with access logs
+        if (!isset($uuid)) {
+            return $this->list_vhosts();
+        }
+        // emulate GET call for a specific log /accesses/uuid
+        else {
+            return $this->call_configd('access', $uuid);
+        }
+    }
+
+    public function errorsAction($uuid = null) {
+        $this->nginx = new Nginx();
+        // emulate rest API -> /errors delivers a list of servers with error logs
+        if (!isset($uuid)) {
+            return $this->list_vhosts();
+        }
+        // emulate GET call for a specific log /errors/uuid
+        else {
```

Comment:
wrong line warp, should probably be (PSR)
```
if {
} else {
}
```

Reply:
then I'll have to move the comment inside